js/lib/beautify-html.js

```diff
@@ -524,7 +524,7 @@
                     matched = false;
 
                 this.pos = start_pos;
-                input_char = this.input.charAt(this.pos);
+                var input_char = this.input.charAt(this.pos);
```

Comment:
If the file followed that convention it would make sense to follow it. However neither this file nor the project as a whole appear follow the convention.